config/index.js

```diff
@@ -472,6 +474,25 @@ var conf = convict({
       default: true,
       env: 'SECURITY_HISTORY_ENABLED'
     }
+  },
+  lastAccessTimeUpdates: {
+    enabled: {
+      doc: 'enable updates to the lastAccessTime session token property',
+      format: Boolean,
+      default: false,
+      env: 'LASTACCESSTIME_UPDATES_ENABLED'
+    },
+    sampleRate: {
+      doc: 'sample rate for updates to the lastAccessTime session token property, in the range 0..1',
+      format: Number,
+      default: 1,
+      env: 'LASTACCESSTIME_UPDATES_SAMPLE_RATE'
+    },
+    enabledEmailAddresses: {
+      doc: 'regex matching enabled email addresses for updates to the lastAccessTime session token property',
+      default: '.+@mozilla\\.com$',
```

Comment:
This was intentional, so let me run through my thinking.

Ideally, I'd like this entry to have a `format: RegExp` line so that convict constructs the `RegExp` instance on our behalf. Right now, that instantiation happens in every call to `features.isLastAccessTimeEnabled` instead, which is obviously inefficient. I would have cached it when we read the config, except the tests depend on being able to change it mid-flight.

What I'd rather do is add support for `RegExp` to convict, then come back and add the format to our config. Then we can avoid paying to construct the `RegExp` on every call both for this feature and for sign-in confirmation, with the tests just passing in a fully-constructed `RegExp` instance instead of a string.

Additional to that, I see the use of an array for sign-in confirmation as a historical accident from when [it was an array of strings rather than an array of regexes](https://github.com/mozilla/fxa-auth-server/pull/1275/files#diff-c8c5ee8e781f831e4927a392fa3565e6R426). We can express everything we want to from an array as a single regex anyway, and I can't see a likely future where one regex for all supported email addresses becomes so unwieldy as to warrant an entire array of them.

So if anything, once I've added `RegExp` support to convict, I'd rather come back and update the config for sign-in conformation to make that conform with what I've done here.

How does that sit with you chaps?